web/src/components/netflow-traffic.tsx

```diff
@@ -1451,7 +1453,11 @@ export const NetflowTraffic: React.FC<{
             <FlowsQuerySummary
               flows={flows}
               stats={stats}
+              loading={loading}
               lastRefresh={lastRefresh}
+              lastDuration={lastDuration}
+              warningMessage={warningMessage}
+              slownessReason={slownessReason()}
```

Comment:
Something else I don't explain just looking at the code: for some reason I see the slowness message working for metrics query summary but not for flow query summary. You can see here, query took 9s but no warning:
![Capture d’écran du 2024-01-11 11-31-07](https://github.com/netobserv/network-observability-console-plugin/assets/2153442/67b2d853-2b3a-46b2-a247-ae4cc7ccdabe)

Reply:
Ok, I think there is a totally different issue, probably not due to this PR, that is preventing the `Promise.race` call to work as expected when the flows table is displayed. In fact, when flow table is loading, the whole UI seems barely usable, like frozen. For instance, I can't open the query options or filters while it's fetching data. In topology or overview I don't have this problem, the UI is reactive. I have the feeling something is taking all compute time and doesn't give a chance to the `setTimeout` promise used in `race` to run properly. I can reproduce that problem both in chrome and firefox.

Reply:
Do you have all columns displayed ? That's typically something related to view drawing performances

Reply:
Table is having an issue rendering all rows. I have fixed it in https://github.com/netobserv/network-observability-console-plugin/pull/438/commits/b7a0939fa4a16e458d9e8f10158f884d68bbc865

Reply:
yep that sounds to have fix it! thanks